django_project/feti/models/feedback.py

```diff
@@ -1,49 +1,62 @@
 # coding=utf-8
 """Model class for feedback"""
 
-from django.db import models
-from django.utils import timezone
+from django.contrib.gis.db import models
+import datetime
```

Comment:
@ann26 could you rearrange here?
the arragement is
- python library
- django library
- third party
- our project

and arrange alphabetically
```
import datetime
from django.contrib.gis.db import models
from django.utils.translation import ugettext_lazy as _
```